services/vod-to-live/service/src/domains/live-stream/prepare-channel-live-stream.ts

```diff
@@ -38,20 +52,37 @@ export const prepareChannelLiveStream = async (
           },
         );
       }
+      const storedChannelJson = await storage.getFileContent(
+        generateChannelFilePath(channelId, metadataFileName),
+      );
+      const storedChannelMetadata: ChannelMetadataModel =
+        JSON.parse(storedChannelJson);
+      newChannelMetadata.key_id = storedChannelMetadata.key_id;
     } else {
+      const channelKey = await keyServiceApi.postContentKey(channelId);
+      newChannelMetadata.key_id = channelKey.Id;
+      const protectionCpixCreationResult = await createProtectionCpix(
+        channelKey.Id,
+        channelId,
+        keyServiceApi,
+        storage,
+      );
       const channelCreationResult = await virtualChannelApi.putChannel(
         channelId,
         smil,
       );
+      //TODO: send RabbitMQ event message to channel's key_id (or send channel metadata + key_id?)
```

Comment:
What is the TODO here? Send which message to a channel key id (would that be a special routing key)?